src/types/markers.js

```diff
@@ -130,7 +130,7 @@ export type GCMajorCompleted = {
   // The number for the start of this GC event.
   major_gc_number: number,
   minor_gc_number: number,
-  slice_number: number,
+  slice_number?: number,
```

Comment:
Nit: can you add a comment explaining why it's optional ? (as I understand it, it's because it's missing in older profiles ?)